OntoTrans2ndWorkshop/README.md

```diff
@@ -0,0 +1,286 @@
+
+# Instructions for setup and requirements
+
+In this example, YAML files are used to specify a declarative workchain based on AiiDA shell.
+Those files contain references to local files that need be copied to a remote folder for execution.
+In the current version, absolute paths are used.
+To make those paths system-independent (*i.e.* to allow the workflow being executed on different machines), the paths have been renamed to `/tmp/OntoTrans2ndWorkshop`.
+Wherever your local repo is located, issue these commands prior to execution in order to create a softlink to your OntoTRans2ndWorkshop folder.
+Note that the lammps executable in workflow_full.yml needs to be set to you lammps installation.
+
+```bash
+$ cd /tmp
+$ ln -s /path/to/OntoTrans2ndWorkshop
+```
+
+The minimal requirement to run this example is:
+- [ExecFlow](https://github.com/H2020-OpenModel/ExecFlow)
+  (Which requires AiiDA-core, oteapi-core, oteapi-dlite, DLite-Python, Tripper, AiiDA-shell)
+and a simlified lammpsparser.
+
+This can all be installed by running `pip install .` in the current folder
```

Comment:
Please also add a link to how to install and setup AiiDA: https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/get_started.html

Maybe with a quick summary, like
- install postgresql and rabbitmq (ex: `sudo apt install git postgresql postgresql-server-dev-all postgresql-client rabbitmq-server`)
- create virtual environment (ex: `python -m venv ~/envs/aiida && source ~/envs/aiida/bin/activate`) 
- install Python packages (ex: `pip install .`)
- setup profile (ex: `verdi quicksetup`) 
- start verdi deamon (ex: `verdi daemon start`)

Reply:
I can add a link to the how to. I disagree with a full documentation on how.It is duplication of things that quickly end up being outdated.

Reply:
I was only thinking about a quick summary. This if fine